android/forms-screenlet-demo/app/src/main/res/layout/app_bar_main.xml

```diff
@@ -21,9 +22,9 @@
 				android:background="@color/white"
 				android:elevation="@dimen/toolbar_elevation"
 				android:fitsSystemWindows="true"
-				android:title="@string/insurances"
 				android:titleTextColor="@color/toolbar_text_color"
 				android:theme="@style/AppTheme.PopupOverlay"
+				app:title="@string/insurances"
```

Comment:
Because in this case when you use v7 Toolbar you can't change the toolbar attributes using android.
https://stackoverflow.com/questions/26852108/how-do-you-set-the-title-color-for-the-new-toolbar